examples/print_tree.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+'''
+This script demonstrates using the AtramhasisProvider to print a full thesaurus
+as a simple rst tree. The generated tree can be saved as a file and turned into
+a pdf or imported in a python Sphinx documentation project.
+'''
+
+from skosprovider_atramhasis.providers import AtramhasisProvider
+from skosprovider.providers import DictionaryProvider
+from skosprovider.utils import dict_dumper
+
+import requests
+
+# Use to only generate the tree for certain top terms.
+# TOP_CONCEPTS = None
+TOP_CONCEPTS = [359]
+
+def title(title, underline='='):
+    print(title)
+    print(underline * len(title))
+
+def list(items, provider, indent=''):
+    print()
+    for i in items:
+        if i['type'] == 'collection':
+            tpl = '%s* <`%s <%s>`_>' if i['type'] == 'collection' else '%s* `%s <%s>`_'
+            print(tpl % (indent, i['label'], i['uri']))
+        else:
+            tpl = '%s* `%s <%s>`_'
+            print(tpl % (indent, i['label'], i['uri']))
+
+        child = provider.get_children_display(
+            i['id'],
+            language='nl-BE',
+            sort='label'
+        )
+        if (len(child)):
+            list(child, provider, indent=indent + ' ')
+    print()
```

Comment:
Not sure anymore, it's an older script. I assume that without the print, the rst didn't render properly. But I'll test that.

Reply:
I can confirm the empty lines are necessary for the rst2pdf to function properly. Otherwise you get:

```
skosprovider_atramhasis) koen@temeraire:~/Projecten/python/skos/skosprovider_atramhasis$ rst2pdf test.rst
/usr/lib/python3/dist-packages/rst2pdf/rson.py:140: FutureWarning: Possible nested set at position 2
  splitter = re.compile(pattern).split
test.rst:6: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:9: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:10: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:11: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
test.rst:19: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
test.rst:20: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:21: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:22: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
test.rst:25: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
test.rst:34: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
test.rst:36: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
(skosprovider_atramhasis) koen@temeraire:~/Projecten/python/skos/skosprovider_atramhasis$ 
```